docker-compose.yaml

```diff
@@ -55,6 +55,19 @@ services:
       - bioos-storage:/nfs/bioos-storage
     networks:
       - bioos
+  ga4gh-wes:
```

Comment:
Service name should be `ga4gh-wes-nextflow` since it's only backend for nextflow.

Reply:
done.